docs/config_manager/load.md

```diff
@@ -34,84 +34,29 @@ in the `cisco_ios` host group.
 The `config_manager/load` function also supports replacing the current active
 configuration with the configuration file located on the Ansible controller.
 In order to replace the device's active configuration, set the value of the
-`config_replace` setting to `True`.
+`config_manager_replace` setting to `True`.
 
 ```
 - hosts: cisco_ios
 
   roles:
     - name ansible_network.cisco_ios
       function: config_manager/load
-      config_file: files/ios.cfg
-      replace: yes
+      config_manager_text: "{{ lookup('file', 'ios.cfg') }}"
+      config_manager_replace: yes
```

Comment:
The text on line 37 says to set this to 'True', but this example sets it to 'yes'.  Would be better to either set this to 'true' here, or change the text to say 'yes' so text and example match.

Reply:
Fixed @samccann